core/domain/exp_services.py

```diff
@@ -1370,32 +1384,32 @@ def _should_index(exp):
     return rights.status != rights_manager.ACTIVITY_STATUS_PRIVATE
 
 
-def get_average_rating_from_exp_summary(exp_summary):
-    """Returns the average rating of the document as a float. If there are no
+def get_average_rating_from_exp_summary(exp_summary_ratings):
```

Comment:
Just call it 'ratings' and get_average_rating, and get rid of the mentions of exp_summary. This function does not need to know the context.